pages/data.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Data
+title: Metadaten
 layout: data
 permalink: /data.html
 ---
 
-## Collection Metadata
+## Metadaten der Sammlung
 
-The table below provides sorting and basic search of the collection contents. 
-Use the "CSV" button below to download the filtered metadata you see on the page. 
-Alternatively, click the "Download" button at the top right to view the full collection metadata in various formats. 
+Die folgende Tabelle ermöglicht die Sortierung und eine einfache Suche nach dem Inhalt der Sammlung.
+Verwenden Sie die Schaltfläche "CSV", um die gefilterten Metadaten herunterzuladen.
+Alternativ können Sie über die Schaltfläche "Herunterladen" oben rechts klicken, um die vollständigen Metadaten der Sammlung in verschiedenen Formaten anzuzeigen.
```

Comment:
Maybe the button "Download" should be translated differently, as you click it not to download data, but to get a list of more formats you can then download.
For example: "Weitere Formate" or "Other Download Options" etc.

Reply:
It's not perfect, but it won't be the most widely used button on this page ;) Additionally: I am hesitant to change it as it would require the upstream repo to change the button as well.